apps/docs/content/docs/components/rating.mdx

```diff
@@ -0,0 +1,175 @@
+| description        | `ReactNode`                                                                                                                                                            | A description for the rating. Provides a hint such as specific requirements for what to choose.                                                                | -             |
+| errorMessage       | `ReactNode`                                                                                                                                                            | A errorMessage for the rating.                                                                                                                                 | -             |
+| isRequired         | `boolean`                                                                                                                                                              | Whether user rating is required on the input before form submission.                                                                                           | `false`       |
+| isReadOnly         | `boolean`                                                                                                                                                              | Whether the rating can be selected but not changed by the user.                                                                                                |               |
+| isDisabled         | `boolean`                                                                                                                                                              | Whether the rating is disabled.                                                                                                                                | `false`       |
+| isInvalid          | `boolean`                                                                                                                                                              | Whether the rating is invalid.                                                                                                                                 | `false`       |
+| baseRef            | `RefObject<HTMLDivElement>`                                                                                                                                            | The ref to the base element.                                                                                                                                   | -             |
+| validationState    | `valid` \| `invalid`                                                                                                                                                   | Whether the rating should display its "valid" or "invalid" visual styling. (**Deprecated**) use **isInvalid** instead.                                         | -             |
```

Comment:
I don't think it's necessary to include deprecated props in the new component. Please remove it.